common/SpotifyHelper.spec.ts

```diff
@@ -26,7 +26,7 @@ describe("SpotifyHelper", () => {
             fetch.mockRejectOnce(new Error("foo"))
             let message = ""
             try {
-                await helper.get("whatever")
+                await helper.get("whatever").then((response) => (message = response.message))
             } catch (e) {
                 message = e.message
             }
```

Comment:
yeah, so because i changed the get function to return errors instead of throwing them, for some reason the error comes through in the then instead of catch.